src/java.desktop/share/classes/javax/swing/text/html/HTMLEditorKit.java

```diff
@@ -1182,6 +1181,15 @@ public void handleEndOfLineString(String eol) {
      * A factory to build views for HTML.  The following
      * table describes what this factory will build by
      * default.
+     * <p>
+     * Parsed tags that lack full support are handled differently by the editor.
+     * When the container using HTML tags is editable, all non-supported tags
+     * will be displayed as editable text fields in place of the tags. The contents
+     * within the tags will be handled by the editor as regular text.
+     * If the container is not editable and a tag is recognized but not supported,
+     * such as script tags, the tag and its contents will be hidden.
+     * If the container is not editable and a tag is unknown or not supported at
+     * all, the tags will be hidden but its contents will display as regular text.
```

Comment:
"unknown or not supported at all" ?
unknown tags are surely always unsupported so why do we need to say that here ?

With all the above it ends up something like 
     * @implNote
     * Parsed tags that are unrecognized or are recognized but unsupported are handled differently by the editor.
     * <ul>
     * <li> When the container using HTML tags is editable, all such tags
     * The contents will be displayed as editable text fields in place of the tags.
     * The content within the tags will be handled by the editor as regular text.
     * <li> If the container is not editable and a tag is recognized but not supported,
     * such as script tags, the tag and its contents will be hidden.
     * <li> If the container is not editable and a tag is unknown
     * the tag will be hidden but its contents will display as regular text.
      *</ul>
     * <p>
     * <p>

Reply:
I took @aivanov-jdk and your advice and reformatted the text into list formatting. I reworded some statements to better differentiate tag text vs contents within the tags. I hope it's clear, but let me know if there is still anything unclear.